content-src/components/Sections/Sections.jsx

```diff
@@ -137,14 +137,21 @@ export class Section extends React.PureComponent {
     const maxCards = maxCardsPerRow * numRows;
     const maxCardsOnNarrow = CARDS_PER_ROW_DEFAULT * numRows;
 
+    const {pocketCta, isUserLoggedIn} = Pocket || {};
+    const {useCta} = pocketCta || {};
+
+    // Don't display anything until we have a definitve result from Pocket,
+    // to avoid a flash of logged out state while we render.
+    const isPocketLoggedInDefined = (isUserLoggedIn === true || isUserLoggedIn === false);
+
     const shouldShowPocketCta = (id === "topstories" &&
-      Pocket.pocketCta.useCta && !Pocket.isUserLoggedIn);
+      useCta && isUserLoggedIn === false);
 
-    // Show topics only for top stories and if it's not initialized yet (so
-    // content doesn't shift when it is loaded) or has loaded with topics
+    // Show topics only for top stories and if it has loaded with topics.
+    // The classs .top-stories-bottom-container ensures content doesn't shift as things load.
     const shouldShowTopics = (id === "topstories" &&
-      (!topics || topics.length > 0) &&
```

Comment:
I noticed the previous code would display empty topics as a placeholder of sorts to ensure we don't have shifting content while the page loads. I added a wrapper `top-stories-bottom-container` and moved the "more recs" bit out of topics so we no longer need to rely on topics for this, thus we can remove the `!topics` bit from the `!topics || topics.length > 0` check.